charts/k8s-monitoring/tests/platform/eks-with-windows/deployments/query-test.yaml

```
@@ -83,6 +85,10 @@ spec:
           - query: count_over_time({cluster="$CLUSTER", job="integrations/kubernetes/eventhandler"}[1h])
             type: logql
 
+          # Node logs
+          - query: count_over_time({cluster="$CLUSTER", job="integrations/kubernetes/journal"}[1h])
+            type: logql
+
           # Pod logs
-          - query: count_over_time({cluster="$CLUSTER", job!="integrations/kubernetes/eventhandler"}[1h])
+          - query: count_over_time({cluster="$CLUSTER", job="default/kube-state-metrics"}[1h])
```

Is it less fragile to exclude both events and node logs?

Mainly thinking about this query from the k8s app perspective and I know it doesn't have much impact here... unless you have a test with a different KSM job label or KSM logging turned off/filtered.

Fine either way, just made me think! 😄 

```suggestion
          - query: count_over_time({cluster="$CLUSTER", job!~"integrations/kubernetes/eventhandler|integrations/kubernetes/journal"}[1h])
```